Source/Core/Common/BitSet.h

```diff
@@ -73,7 +73,7 @@ class BitSet
       else
       {
         int bit = std::countr_zero(m_val);
-        m_val &= ~(1 << bit);
+        m_val &= ~(static_cast<IntTy>(1) << bit);
```

Comment:
I think it looks better this way:
```suggestion
        m_val &= ~(IntTy{1} << bit);
```